tests/lib/acpi/src/main.c

```diff
@@ -8,6 +8,22 @@
 #include <zephyr/kernel.h>
 #include <zephyr/acpi/acpi.h>
 
+#if (ACPI_DT_HAS_HID(DT_NODELABEL(pcie0)))
```

Comment:
What you're really interested in is the same condition that the PCIe code itself uses to require a HID property, i.e. `CONFIG_PCIE_PRT`. I think that's what you should check for here.

Reply:
This test case nothing to do with PRT. It test acpi_current_resource_get() and PCIe chosen since it will be default available. Updated to use acpi_dev and can be selected based on board configuration/overlay (as you suggested early)